### PR TITLE
Overwrites the availability text or the stock info text on the PDP

### PR DESCRIPTION
**Pull request by patrick-scherer**

### CODE REVIEW

File: extension-config.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "version": "1.0.0",
+  "id": "@shopgate-project/ext-static-availability-stockinfo-text",
```
Comment (Carsten89) [inline extension-config.json]:
```suggestion
  "id": "@shopgate-project/static-availability-stockinfo-text",
```

File: extension-config.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    },
+    {
+      "id": "StaticAvailabilityText",
+      "type": "portals",
+      "path": "frontend/portals/StaticAvailabilityText",
+      "target": "product.availability"
+    }
+  ],
+  "configuration": {
+    "apiToken": {
```
Comment (Carsten89) [inline extension-config.json]: why is this in here?

**Review by Carsten89 — CHANGES_REQUESTED** [review]

File: extension-config.json (hunk 2 above)
Comment (patrick-scherer) [reply extension-config.json]: removed

File: extension-config.json (hunk 1 above)
Comment (patrick-scherer) [reply extension-config.json]: done

**Review by Carsten89 — APPROVED** [review]

**Pull request merged**